Assets/MRTK/Providers/OpenVR/MRTK.OpenVR.asmdef

```diff
@@ -1,9 +1,9 @@
 {
     "name": "Microsoft.MixedReality.Toolkit.Providers.OpenVR",
     "references": [
-        "Microsoft.MixedReality.Toolkit"
+        "Microsoft.MixedReality.Toolkit",
+        "Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.Shared"
```

Comment:
I think we might want to put the shared definition into the core instead, matching the location of the current WMR one: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MRTK/Core/Definitions/Devices/WindowsMixedRealityControllerDefinition.cs

Reply:
And that way we don't have this unexpected WMR/OpenVR provider dependency